### PR TITLE
deploy linkerd stable channel

### LINKED ISSUE #448: Deploy from Linkerd's stable channel by default
#### Current Behavior

MeshOps V1 picks up latest version(edge or stable) to deploy Linkerd Service Mesh.
#### Expected Behavior

While using MeshOps V1 where the users do not specify a particular version, prioritise the installation of latest stable over edge.
#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v
- **Meshery Adapter Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 📝 [Meshery Adapters Spreadsheet](https://docs.google.com/spreadsheets/d/1rGGpSXC68iDJzNRU-qZIRQsZUwkt1qLFg7JCs4pfvbU/edit?usp=sharing)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by duckling69**
**Description**

This PR fixes #448 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Running build checks...

**Comment by leecalcote** [comment]
Speedy Gonzales here ⚡

File: .github/workflows/e2etests.yaml [hunk 1]
```diff
@@ -26,15 +26,21 @@ jobs:
...
                 echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
-            else 
-                echo "version=edge" >> $GITHUB_ENV
-            fi      
-      - name: Get latest release tag
-        id: gettag
+            else
+                echo "version=$(steps.getadapter.outputs.release)" >> $GITHUB_ENV
+            fi
+      - name: Get latest stable release of adapter
```
Comment (Aisuko) [inline .github/workflows/e2etests.yaml:32]: Here we want to "Get the latest stable release of Linkerd2"

File: .github/workflows/e2etests.yaml [hunk 2]
```diff
@@ -26,15 +26,21 @@ jobs:
...
+        id: getadapter
         uses: pozetroninc/github-action-get-latest-release@master
         with:
           repository: linkerd/linkerd2
-          excludes: prerelease, draft, edge 
+          excludes: prerelease, draft, edge
```
Comment (Aisuko) [inline .github/workflows/e2etests.yaml:37]: I wonder if this Github action has any cache for the latest release. If yes, please add those parameters to enable the cache.

**Review by Aisuko — CHANGES_REQUESTED** [review]

**Comment by Aisuko** [comment]
By the way, the `golangci-lint` errores are totally not related to this PR

File: .github/workflows/e2etests.yaml (hunk 2 above)
Comment (duckling69) [reply .github/workflows/e2etests.yaml]: It have cache for latest release but the issue talks about latest and stable version couldn't find any cache for that or any github action

Comment (Aisuko) [reply .github/workflows/e2etests.yaml]: @duckling69 Thanks for clarify it

**Review by Aisuko — APPROVED** [review]
LGTM

**Pull request merged**